src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
+        let commandText = '';
+        let command = '';
+
+        if (input.indexOf(' ') !== -1 && input.startsWith('/')) {
```

Comment:
Well, seeing as I did think about it, in that I believe that due to the if it's pretty solid, I'll settle for this PR. =)
But I'd be glad if you wrote a test for it later down the road, I tried today but it seems as though my brain doesn't work. :)